docker/container-test.sh

```diff
@@ -33,3 +33,10 @@ function test_container {
     # Return proper exit code.
     [[ ${RESULT} == '"success"' ]]
 }
+
+function test_javahome {
```

Comment:
This needs to be called from the two `build-*` scripts. Alternatively, is it possible to integrate this into the above `test_container` method so all the tests are in one function?

Reply:
it can certainly be included into the function `test_container()`. I usually put one test case per function.

Reply:
We could rename `test_container` to `test_presto_starts`, then create a new one that runs all the tests
```bash
function test_container {
    test_javahome && test_presto_starts
}
```